File: cpp/sprint1_nonfinals/k.cpp
#include <iostream>
#include <vector>

using namespace std;

vector <int> GetSum(const vector <int> &numberList, int k) {
	// Здесь реализация вашего решения

}

void print(const vector <int> &result) {
	for (int x : result)
		cout << x << " ";
	cout << endl;
}

int main() {
	int n;
	cin >> n;
	vector <int> numberList(n);
	for (int i = 0; i < n; ++i)
		cin >> numberList[i];
	int k;
	cin >> k;
	print(GetSum(numberList, k));
}

File: swift/sprint1_nonfinals/k.swift
func readInts() -> [Int] {
    return readLine()!.split(separator: " ").compactMap { Int($0)! }
}

func readInt() -> Int {
    return Int(readLine()!)!
}

func print(_ result: [Int]) {
    print(result.compactMap { String($0)}.joined(separator: " "))
}

func getSum(_ numberList: [Int], _ k: Int) -> [Int] {
    // Здесь реализация вашего решения
}

func main() {
    let n = readInt()
    let numberList = readInts()
    let k = readInt()
    print(getSum(numberList, k))
}

main()
